css-canvas-medical/index.html

```diff
@@ -0,0 +1,127 @@
+          <p><a href="about-us.html" class="link">About Us</a></p>
+        </div>
+        <div class="navBarlink">
+          <p><a href="about-us.html" class="link">Blog</a></p>
+        </div>
+        <div class="navBarlink">
+          <button id="contactButton">Contact Us</button>
+        </div>
+      </div>
+    </div>
+    <!--Row 1 end-->
+    <div class="emrPic">
+      <div class="overlay"></div>
+      <div class="overlayTextContent">
+        <h1 id="title">The EMR Reimagined for Primary Care</h1>
+        <p id="paragraph">Can't remember the last time you talked with a patient
+          without focusing on the EMR? Canvas lets you rediscover joy in the work you
+          do. Capture meaningful data, identify gaps in care, and move seamlessly
+          between meaningful data within the platform. You stay present with patients
+          while you whole team stays up-to-speed. Be a better doctor with Canvas,
+          designed for primary care providers with the help of primary care providers.</p>
+        <button class="startButton">Get Started</button>
+      </div>
+    </div>
+    <!--Row 2 end-->
+    <div class="cliniciansContainer">
+      <div class="clinicians">
+        <h2>Clinicians Prefer Canvas</h2>
+        <p id="cliniciansText">Physician owned primary care practices across the country have already
+          made Canvas the fabric of their care.</p>
+        <!--Row 3 end-->
+        <div class="locationsContainer">
+          <div class="locations">
+            <p class="city">Cascade Family Practice, Portland OR</p>
+            <p class="city">Temescal Creek Medicine, Oakland CA</p>
+            <p class="city">San Juan Healthcare, Friday Harbor WA</p>
+          </div>
+
+          <div class="locations">
+            <p class="city">Ward Medical Clinic, Bonifay FL</p>
+            <p class="city">Caribou Medical, Towanda PA</p>
+            <p class="city">Bay Tree Family Care, Red Bay AL</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!--Row 4 end-->
+    <div class="learnMoreRow">
+      <div class="learnMoreText">
+        <h2 id="learnMore">Ready to learn more?</h2>
+        <p id="schedule">Schedule a private meeting now</p>
+      </div>
+      <div class="learnMoreButtonContainer">
+        <button id="learnMoreButton">Get Started</button>
+      </div>
+    </div>
+
+    <div class="contactFooter">
+      <div class="col-thirds">
+        <div class="whiteLogoContainer">
+          <img src="images/logo-white.png" class="logo">
+        </div>
+        <div class="contactInfoContainer">
+          <p class="contactInfo">2307 Irving Street, Suite 228</p>
+          <p class="contactInfo">San Francisco CA 94122</p>
+          <p class="contactInfo">(800) 370-1416</p>
+          <p class="contactInfo">info@canvasmedical.com</p>
+        </div>
+      </div>
+      <div class="col-thirds">
+          <ul>
+            <li class="firstLi"><a href="">Company</a></li>
```

Comment:
Try a heading element for each of these lists instead of styling the first list item.